### DISCUSSION TITLE
related to the website

### DISCUSSION THREAD

**Discussion by venom-2** [GameOver2811/Knack-Club#10] [General]
Add the google.png file and also login page is not coming the center.
![Capture](https://github.com/GameOver2811/Knack-Club/assets/103876428/3515323b-d4c6-4c7b-8c2f-b5b788257ad8)

**Comment by anshpandey8368** [comment]
Thanks for your time and effort for recognizing this issue.
Our frontend dev will reach out to this issue and resolve for better UI/UX.

**Comment by GameOver2811** [comment]
Thanks for your feedback!
